src/types/ProposalFieldValue.ts

```diff
@@ -29,6 +36,11 @@ export const proposalFieldValueWriteSchema: JSONSchemaType<ProposalFieldValueWri
 			value: {
 				type: 'string',
 			},
+			isValid: {
+				type: 'boolean',
+			},
 		},
-		required: ['applicationFormFieldId', 'position', 'value'],
+		required: ['applicationFormFieldId', 'position', 'value', 'isValid'],
 	};
+
+export { WritableProposalFieldValue, InternallyWritableProposalFieldValue };
```

Comment:
Since you're switching to this model (which is good, it's the future!)  You should probably remove `export` from the earlier definitions as well and add them to this list explicitly.